enterprise/frontend/src/metabase-enterprise/data-studio/data-model/components/TablePicker/components/Results.tsx

```diff
@@ -483,6 +483,17 @@ const ResultsItem = ({
     }
   };
 
+  // Dedicated toggle target: always toggles expansion without navigating
+  // regardless of whether the item is currently active.
+  const handleChevronClick = (e: React.MouseEvent) => {
```

Comment:
nit: we usually destructure react types imports

```suggestion
  const handleChevronClick = (e: MouseEvent) => {
```